### ISSUE TITLE
acl for optional depictive in embedded infinitival clause

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/docs#855]
- I set myself to attempt the difficult repairs all **alone**

1. Should "alone" be considered a depictive ADJ, or an ADV?
2. Substituting a more canonical depictive adjective, "drunk" gives:
 
   - I set myself to attempt the difficult repairs **drunk**

   Is `acl(I, drunk)` the correct analysis per the [guidelines for optional depictives](https://universaldependencies.org/u/dep/acl.html)? It involves connecting to the subject of the matrix verb. But this feels awkward, and in fact no overt subject is necessary for such a depictive:

   - Attempting the difficult repairs drunk is not recommended.

   Would it not be more accurate to for `acl` to attach to the local verb, i.e. `acl(attempt(ing), drunk)`, and leave the resolution of the predicand to semantic interpretation?

**Comment by dan-zeman** [comment]
As per the current [guidelines on secondary predicates](https://universaldependencies.org/u/overview/complex-syntax.html#secondary-predicates), optional depictives should be attached as `acl` on the nominal if it is present, and `advcl` on the verb if the nominal is not present:

* [en] _He died young._ `acl(He, young)`
* [en] _He ate the steak almost raw._ `acl(steak, raw)`
* [cs] _Vstoupila do místnosti smutná._ "She-entered to room sad." `advcl(Vstoupila, smutná)`

The guidelines do not show examples where the nominal would only be present in a superordinate clause. We could either say that the nominal is present (in the higher clause) and do `acl(I, drunk)`; or say that it is not present (because we want it in the lower clause) and do `advcl(attempt, drunk)`. I suspect that the two alternative attachments could be understood as two different readings of the sentence. Let's say the sentence is

* I want to do the repairs drunk.

If I am drunk during the wanting event, the _drunk_ belongs to the higher clause and should be attached to _I_. If I am now sober but I want to first get drunk and then do the repairs, then _drunk_ belongs to the event of doing repairs, i.e., to the lower clause.

**Comment by nschneid** [comment]
> As per the current [guidelines on secondary predicates](https://universaldependencies.org/u/overview/complex-syntax.html#secondary-predicates)

Thanks for the pointer—it would help if `acl` and `xcomp` guidelines linked to that.

I'm afraid that the guidelines for depictives yield what I would consider counterintuitive results for English (maybe they make more sense for other languages). They are confusing enough that I have to double-check every time I encounter a depictive to make sure I am getting it right. Some reasons they strike me as problematic:

(a) The `acl` approach creates flagrant non-constituents. ("I will not perform drunk"—"I"+"drunk" is obviously not a constituent. Same goes for "I painted the model naked"—"model naked" is not a constituent.) I know, UD does not have a formal notion of constituency, but from an English annotation perspective there should be a high bar for this sort of thing.

(b) It is not specified what to do if the nominal is nonlocal to the clause with the depictive—acl/nominal attachment or advcl/clause attachment?:

- **I** am reluctant to perform **drunk**
- **my** reluctance to perform **drunk**

As @dan-zeman notes above, the clause the depictive belongs to can be ambiguous:

- I **arrived** to **perform** drunk
   ... but fortunately they stopped me and sent me home. [higher clause]
   ... but there was no alcohol available so I had to perform sober. [lower clause]

So in such cases we are forced to choose between using the dependency to disambiguate the clause and using it to disambiguate the nominal predicand.

(c) It is not specified what to do if there are multiple plausible predicand nominals in the clause:

- **I** found **myself** drunk

(d) The split policy of `acl` if the nominal is overt, `advcl` if it is not suggests they are different constructions, but I don't think this is the case:

- wishing to arrive **drunk**
- wishing that I could arrive **drunk**

The difference here is in the kind of subordinate clause and whether it requires an overt subject, not really in the way the depictive adds on to the clause or is interpreted.

(e) Consider also the phrase order variation like

- I came in out of the rain **sopping wet**.
- I, **sopping wet**, came in out of the rain.
- **Sopping wet**, I came in out of the rain.

For simplicity of annotation, it would be nice if these were all handled the same.

In sum, at least for English, it seems like the nominal that the depictive describes is a matter of semantic interpretation, not a surface-structural dependency. Having all optional depictives attach to the clause as `advcl` would be simpler and more consistent. It also parallels the treatment of mandatory depictives and resultatives, which attach to the clause as `xcomp`.

**Comment by nschneid** [comment]
On a practical level, in EWT this construction is not terribly frequent, nor are the annotations consistent: [this query](http://universal.grew.fr/?corpus=UD_English-EWT@dev&custom=624860afbe90e) finds 18 matches with `acl` spanning across a non-participial VERB (not all of these are depictives); 3 more tokens [here](http://universal.grew.fr/?corpus=UD_English-EWT@dev&custom=624869cec996f); [this query](http://universal.grew.fr/?corpus=UD_English-EWT@dev&custom=6248623689f86) finds 17 non-copular ADJs attaching as `acl` (again, not all are depictives). For ADJs attaching as `advcl`, [this query](http://universal.grew.fr/?corpus=UD_English-EWT@dev&custom=624863039c0f5) gives 22 matches.

**Comment by sylvainkahane** [comment]
Linking the adjective to the noun is a very semantically motivated analysis. As pointed by @nschneid, they do not form a constituent and therefore should not linked by a syntactic dependency. I can add more data:

- the noun can be passived: _the model was painted naked_
- the noun can be extracted: _a model that I painted naked_
- the noun can be pronominalized. In French it is a preverbal clitic: _je l'ai peint nu_ , lit. 'I it painted naked'.

It seems clear that that the adjective raises on the verb. I have no clear intuition for the relation name: if `acl` vs `xcomp` corresponds to modifier vs argument, `acl` seems good.

**Comment by amir-zeldes** [comment]
In the Stanford Dependencies days, we used to have these as advcl on the predicate (i.e. in GUM <5, before the switch to UD). We understood it as having something like an elliptical 'being' or similar. I'm not 100% sure if this was the intention of the Stanford guidelines, but it was what CoreNLP mainly did with these things in English, which we copied. I was honestly pretty happy with that analysis and changed to the adnominal `acl` somewhat reluctantly, in order to conform to UD guidelines. I agree with @sylvainkahane that this analysis doesn't seem syntactically motivated for English. If there is willingness to change it, I would say the starting point could maybe be a return to advcl, basically saying the following have the same dependent clause parent and child:

  * I painted the model naked - advcl(painted, naked) - this may seem odd considering the current guidelines, but consider the following:
  * I painted the model while he was naked - advcl(painted, naked) - this creates parity with/without a complementizer
  * a model that I painted naked - advcl(painted, naked) - parity with relative cases
  * a model that I painted while he was naked - advcl(painted, naked) - and both combined

Basically this treats "naked" as short for "while X is/was naked".

**Comment by dan-zeman** [comment]
> In sum, at least for English, it seems like the nominal that the depictive describes is a matter of semantic interpretation, not a surface-structural dependency.

In Czech it can be surface-structural as well:

- _Potkala syna smutná_ "She-met son.ACC sad.NOM" (She was sad when meeting son.)
- _Potkala syna smutného_ "She-met son.ACC sad.ACC" (The son was sad when she met him.)
  - This is different from _Potkala smutného syna_ where the adjective is a regular `amod` of the object. Congruent attributes normally precede the modified noun in Czech.

**Comment by nschneid** [comment]
Interesting, so the relevant predicand is indicated with case agreement? The dependency still has to be `advcl` (not `acl`) in those examples right, because there is no nominal referring expression separate from the verb?

I don't know enough about Czech to draw a conclusion about modification structure—do ordinary amod adjectives also exhibit case agreement with the nominal head? Does case agreement generally imply a direct dependency relation, or could it be more of an anaphoric relation?

**Comment by daghaug** [comment]
In Greek and Latin too, the predicand is indicated with case agreement. This generally does not happen with anaphoric relations in those languages. In our LFG-inspired source annotation, they are `xadj` depending on the verb, but with a secondary dependency like an `xcomp`. But the Stanford Dependencies, and therefore UD, took only `xcomp` and not `xadj` from LFG. Within UD, I agree `advcl` is better because it can be applied whether the nominal is present or not. It also opens the possibility to indicate the predicand with an enhanced edge whenever present. The enhancement guidelines would probably have to be modified, but in my experience people already use `nsubj` EUD edges in much "looser" cases of control.

**Comment by dan-zeman** [comment]
> Interesting, so the relevant predicand is indicated with case agreement? The dependency still has to be `advcl` (not `acl`) in those examples right, because there is no nominal referring expression separate from the verb?

In the nominative example, the nominal is missing (because inferrable from the form of the verb), so yes, the relation has to be `advcl` on the verb. In the accusative example, the nominal is present (_syna_), so it can depend on the nominal, but since it is considered secondary predication, it will be `acl` (while it would be `amod` if the word order is _smutného syna_). I cannot say I find this outcome great because the borderline between `amod` and `acl` stays blurry (it is sometimes possible to put an `amod` after the noun, and generally it seems misguided to categorize dependencies by word order in a language like Czech).


> I don't know enough about Czech to draw a conclusion about modification structure—do ordinary amod adjectives also exhibit case agreement with the nominal head?

Yes.

> Does case agreement generally imply a direct dependency relation, or could it be more of an anaphoric relation?

Most of the time you find it in direct dependency relations (read: I cannot think of a counter-example but I hesitate to claim it does not exist). It's because the case of a nominal is typically required by a verb, preposition, or another nominal, then the case agreement holds within the nominal. Anaphoric relations would have gender and number agreement but not case agreement.

**Comment by Stormur** [comment]
In our Latin treebanks we are uniformly using `advcl` with the subtype `pred` for all these cases (see [page in the documentation](https://universaldependencies.org/la/dep/advcl-pred.html)).

Major points:
* as subjects are not always present/expressed, the relation cannot depend on them;
* the phenomenon is always the same and the distinctions between "depictive" and else as still seen in the general guidelines are particularly problematic, especially because they seem to be based on criteria that are not really proper of UD. So, just one label should be used; 
* the label is `advcl` and not `acl` because all these kinds of predications are expanding nuances of the predication, not of the subject; they are not even limited to the subjects.

**Comment by amir-zeldes** [comment]
> as subjects are not always present/expressed, the relation cannot depend on them;

That's a really good point, in pro-drop languages you'd get a very different tree depending on whether an overt subject was realized...

**Comment by dan-zeman** [comment]
> > as subjects are not always present/expressed, the relation cannot depend on them;
> 
> That's a really good point, in pro-drop languages you'd get a very different tree depending on whether an overt subject was realized...

Exactly. That's what the examples in the guidelines are showing even now (for Czech).

**Comment by amir-zeldes** [comment]
OK, if I'm reading the vibe here correctly, then this is perhaps a candidate for discussing an amendment to the universal guidelines, right?

**Comment by nschneid** [comment]
I guess another consideration here is whether nominalizations can license depictives, and if so how to treat them.

- He performed drunk: `advcl(performed, drunk)` if we change the guidelines
- ?His performance drunk caught me by surprise

If this is possible would it be `acl(performance, drunk)`? That would parallel GUM's treatment of "years active", but most adjectives that are post-noun dependents are `amod`.

**Comment by nschneid** [comment]
@dan-zeman points out that true adverbial clauses are similar:

- She arrived while being happy
- She arrived happy

In both cases there is an understood subject of "happy", and perhaps this should be annotated in the Enhanced layer.

**Comment by msklvsk** [comment]
Related: https://github.com/UniversalDependencies/docs/issues/476.
